module/Photo/view/partial/tags-new.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+$this->scriptUrl()->requireUrl('photo/photo/tag/remove', ['photo_id', 'lidnr'])
+                  ->requireUrl('member/view', ['lidnr']);
+?>
+<?php if($this->acl('photo_acl')->isAllowed('tag', 'view')): ?>
+    <strong><?= $this->photo->getAlbum()->getName() ?></strong>
+    <?php $tagCount = $this->photo->getTags()->count(); $i = 1; ?>
+    <?= $tagCount ? $this->translate(', in this photo:') : '' ?>
+    <div class="tagList tagList-<?= $this->photo->getId() ?>">
+        <?php foreach ($this->photo->getTags() as $tag): ?>
+            <span>
```

Comment:
Add `display: block` to the SPAN elements. This way the remove icons don't get pushed on a line new for smaller screens.